### PR TITLE
Added the apps Astah SysML, Astah Professional and PDFExpert

### PR DESCRIPTION
**Pull request by hutschen**

### CODE REVIEW

**Comment by adityadalal924** [comment]
```
Offenses:
Casks/astah-professional.rb:14:4: C: Final newline missing.
end
   
Casks/astah-sysml.rb:14:4: C: Final newline missing.
end
   
2868 files inspected, 2 offenses detected
RuboCop failed!
```

**Comment by hutschen** [comment]
I added the missing newlines.

File: Casks/pdfexpert.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'pdfexpert' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://readdle-test-binaries.s3.amazonaws.com/release/PDFExpert.dmg'
```
Comment (adityadalal924) [inline Casks/pdfexpert.rb]: Needs a line above `url` -> `# amazonaws.com is the official download host per the vendor homepage`

Comment (hutschen) [inline Casks/pdfexpert.rb]: I inserted that comment with commit 9ece1c2

**Comment by adityadalal924** [comment]
Thank you for the contribution. It was merged directly as c28b0c7db7fdd20246f9ae4f4dec04f3c7452ba2 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**